flask_praetorian/base.py

```diff
@@ -321,24 +325,30 @@ class methods and attributes necessary to function correctly.
         if dummy_user:
             PraetorianError.require_condition(
                 hasattr(dummy_user, "identity"),
-                textwrap.dedent("""
+                textwrap.dedent(
+                    """
                     Instances of user_class must have an identity attribute:
                     user_instance.identity -> <unique id for instance>
-                """),
+                """
+                ),
             )
             PraetorianError.require_condition(
                 self.roles_disabled or hasattr(dummy_user, "rolenames"),
-                textwrap.dedent("""
+                textwrap.dedent(
+                    """
                     Instances of user_class must have a rolenames attribute:
                     user_instance.rolenames -> [<role1>, <role2>, ...]
-                """),
+                """
```

Comment:
please indent this (and all following cases)

Reply:
I hope I didn't leave one out